src/houses/KM_HouseBarracks.pas

```diff
@@ -277,11 +279,24 @@ procedure TKMHouseBarracks.CreateRecruitInside(aIsMapEd: Boolean);
 end;
 
 
+procedure TKMHouseBarracks.ValidateNUpdateRallyPoint;
```

Comment:
I suggest calling this just "ValidateRallyPoint". Validate implies it will fix it if it is invalid, so I don't think Update is necessary. Same for woodcutter.